backend/app/training/find_sitemap.py

```
@@ -0,0 +1,110 @@
+import requests
+from bs4 import BeautifulSoup
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)